.travis.yml

```diff
@@ -1,4 +1,5 @@
-sudo: false
+sudo: required
+dist: trusty
```

Comment:
Right, previously we supposed host Linux systems with old versions of glibc.  This patch would change that and drop support for some currently supported configurations.  That might be the right thing to do, but it's something to do with consideration.